model/file/registry.yaml

```
@@ -111,6 +111,22 @@ groups:
           The user ID (UID) or security identifier (SID) of the file owner.
         stability: experimental
         examples: ["1000"]
+      - id: file.origin_referrer_url
+        type: string
+        brief: >
+          The URL of the webpage that linked to the file.
+        note: > 
+          This information is inteded to be retrieved from the Mark of the Web (NTFS Zone.Identifer ADS Stream) Note that the URL might contain sensitive information.
```

```suggestion
          This information is inteded to be retrieved from the `Mark of the Web` (NTFS Zone.Identifer ADS Stream). Note that the URL might contain sensitive information.
```

also is this stream available only for windows? I know that macos can also contain this data, should we generalise this field?